railties/test/commands/log_test.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+require "rails/commands/log/log_command"
+
+class Rails::Command::LogTest < ActiveSupport::TestCase
+  include ActiveSupport::Testing::Isolation
+  setup :build_app
+  teardown :teardown_app
+
+  test "`rails log:clear` clears all environments log files by default" do
+    Dir.chdir(app_path) do
+      File.open("config/environments/staging.rb", "w")
+
+      File.write("log/staging.log", "staging")
+      File.write("log/test.log", "test")
+      File.write("log/dummy.log", "dummy")
+
+      rails "log:clear"
+
+      assert_equal 0, File.size("log/test.log")
+      assert_equal 0, File.size("log/staging.log")
+      assert_equal 5, File.size("log/dummy.log")
+    end
+  end
+
+  test "`rails log:clear all` clears all log files" do
+    Dir.chdir(app_path) do
+      File.open("config/environments/staging.rb", "w")
+
+      File.write("log/staging.log", "staging")
+      File.write("log/test.log", "test")
+      File.write("log/dummy.log", "dummy")
+
+      rails "log:clear", "all"
+
+      assert_equal 0, File.size("log/test.log")
+      assert_equal 0, File.size("log/staging.log")
+      assert_equal 0, File.size("log/dummy.log")
+    end
+  end
+
+  test "`rails log:clear test` clears only logs for the specified environment" do
```

Comment:
For clarity, let's use a placeholder here like:

```suggestion
  test "`rails log:clear some_environment` clears only logs for the specified environment" do
```

Reply:
I've changed the test to also test the passing multiple environments.
I hope the following is clear enough:
```ruby
  test "`rails log:clear env1,env2` clears only logs for the specified environments" do
```